src/node_options.cc

```diff
@@ -382,6 +377,11 @@ PerProcessOptionsParser::PerProcessOptionsParser() {
   AddOption("--v8-options",
             "print V8 command line options",
             &PerProcessOptions::print_v8_help);
+  AddOption("--prof-process",
+            "process V8 profiler output generated using --prof",
+            &PerProcessOptions::prof_process);
+  // Options after --prof-process are passed through to the prof processor.
+  AddAlias("--prof-process", {"--prof-process", "--"});
```

Comment:
I don’t understand why this is being moved – running `--prof-process` doesn’t affect per-process state, does it?

Reply:
@addaleax this is along the lines of `--help` and `--completion-bash` - if those are in the per-process arguments, `--prof-process` should be in there as well, or otherwise they should all be moved to per environment options.

Reply:
Yeah, I think I’d prefer to have them all be moved to per-env options then…